### ISSUE TITLE
document and maintain declartive tool for creating new kubernetes.io aliases

### ISSUE THREAD

**Issue by philips** [kubernetes/steering#121]
There have been a number of requests for creating new @kubernetes.io aliases to steering because only steering has administrative control of the domain.

However, we need to document when to use kubernetes.io aliases over `kubernetes-*@googlegroups.com` and create a tool to manage the groups over time similar to what the k8s-infra team is doing.

Some of the reasons for `@kubernetes.io` aliases I am aware of:

- googlegroups.com can't nest lists inside of each other, useful for things like aliases where a number of teams need to work together like security@kubernetes.io with release managers and security committee membership.
- Google Cloud can't use @googlegroups.com for access control

**Comment by dims** [comment]
Ack.

/assign @dims

**Comment by philips** [comment]
the blog@ alias had an issue of being configured with only one owner and moderation settings causing the list not to work. It had to be escalated to me to configure and I didn't really know who to give Owner access to because there was no real documentation on how that alias should be managed.

tl;dr we need a declarative approach to this

**Comment by nikhita** [comment]
We have this in place today - https://github.com/kubernetes/k8s.io/blob/master/groups/groups.yaml. Thanks @dims and @cblecker for all the hard work! :rocket: 

IMO this can be closed.
@dims or @cblecker - can you confirm?

**Comment by dims** [comment]
confirmed :)

/close
